src/components/NcSelect/NcSelect.vue

```diff
@@ -972,6 +971,7 @@ export default {
 				calculatePosition: this.localCalculatePosition,
 				filterBy: this.localFilterBy,
 				label: this.localLabel,
+				...this.$attrs,
```

Comment:
@ShGKme I don't know why, but without this it does not work.

Reply:
Because of this undocumented behavior...

https://github.com/vuejs/core/pull/2011#issuecomment-685246268

If a component has some prop (`foo`), then an update event (`update:foo`) is not inherited by root component node, although it is not declared in `emits` and is a part of `$attrs`.

So in this case, a handler in attribute `onUpdate:modelValue` is passed neither as attr, nor as a component event listener to `VueSelect`, because there is a `modelValue` prop. 😣

Reply:
Anyway, we should not pass all the `this.$attrs` here, but implicitly define and re-emit `update:modelValue` event, because only the `update:modelValue` is a problem, not all possible attrs.

Otherwise, there is a duplicating of attrs which is a problem for event handlers.

```vue
<NcSelect @open="handler" />
```

`handler` is called twice.

Reply:
Interesting, thanks for figuring this out. I will have a look how to circumvent this.

Reply:
> Interesting, thanks for figuring this out. I will have a look how to circumvent this.

We only need to define `update:modelValue` event in `emits` and manually proxy it by `@update:modelValue="$emit('update:modelValue', $event)"`

Reply:
Adjusted it. Seems to work fine now.